build.gradle

```diff
@@ -11,8 +11,11 @@ sourceCompatibility = JavaVersion.VERSION_1_6
 dependencies {
   compile 'io.dropwizard.metrics:metrics-core:3.1.0'
   compile 'com.newrelic.agent.java:newrelic-api:3.10.0'
-
+  compile 'com.google.guava:guava:18.0'
```

Comment:
To be friendlier to people who may not want this (effectively optional) dependency, let's put this code in another subproject so that people who don't want this feature don't need to deal with excluding guava, etc.